strawberryfields/decompositions.py

```diff
@@ -1021,53 +1011,7 @@ def bloch_messiah(S, tol=1e-10, rounding=9):
     if np.linalg.norm(np.transpose(S) @ omega @ S - omega) >= tol:
```

Comment:
there is a `is_symplectic` function in the walrus that you can use directly here :)

Reply:
oh! comments on unmodified lines 🧐

(thanks for the suggestion, will do!)